src/mbgl/util/offscreen_texture.hpp

```diff
@@ -9,21 +9,16 @@ class Context;
 class Texture;
 } // namespace gl
 
-enum class OffscreenTextureAttachment {
-    None,
-    Depth,
-};
-
 class OffscreenTexture {
 public:
     OffscreenTexture(gl::Context&,
-                     Size size = { 256, 256 },
-                     OffscreenTextureAttachment type = OffscreenTextureAttachment::None);
```

Comment:
As per above, instead of passing in a type here, we could optionally pass in a `gl::Renderbuffer<gl::RenderbufferType::DepthComponent>` reference (or rather have two constructors, one with and one without).